Src/GoogleApis.Auth.DotNet4.Tests/OAuth2/DefaultCredentialProviderTests.cs

```diff
@@ -192,6 +192,9 @@ public void SetUp()
             Assert.IsInstanceOf(typeof(ServiceAccountCredential), scopedCredential.UnderlyingCredential);
             Assert.IsFalse(scopedCredential.IsCreateScopedRequired);
             CollectionAssert.AreEqual(scopes, ((ServiceAccountCredential) scopedCredential.UnderlyingCredential).Scopes);
+
+            var scopedCredential2 = credential.CreateScoped("scope1", "scope2");
```

Comment:
Done:
1. only using one scopedCredential variable.
2. I split out the tests into two tests